<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/ns_icon.png</normaloff>:/icons/ns_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPanes">
    <property name="title">
     <string>&amp;Panes</string>
    </property>
    <addaction name="actionSplitVertically"/>
    <addaction name="actionSplitHorizontally"/>
    <addaction name="actionKillPane"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionShowConnectivity"/>
    <addaction name="actionShowNoHierarchyEntities"/>
    <addaction name="actionShowEntitiesName"/>
   </widget>
   <widget class="QMenu" name="menuEvents">
    <property name="title">
     <string>&amp;Events</string>
    </property>
    <addaction name="actionAutoPublishSelection"/>
    <addaction name="actionPublishSelection"/>
    <addaction name="actionAutoFocusOnDisplayed"/>
    <addaction name="actionFocusOnDisplayed"/>
    <addaction name="actionFocusOnSelection"/>
    <addaction name="actionAutoFocusOnSelection"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionRecorder"/>
    <addaction name="actionAdvancedRecorderOptions"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCleanScene"/>
    <addaction name="actionJSONImporter"/>
    <addaction name="actionJSONExporter"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPanes"/>
   <addaction name="menuView"/>
   <addaction name="menuEvents"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionShowConnectivity"/>
   <addaction name="actionShowNoHierarchyEntities"/>
   <addaction name="actionShowEntitiesName"/>
   <addaction name="actionCleanScene"/>
   <addaction name="separator"/>
   <addaction name="actionLayouts"/>
   <addaction name="actionStoredSelections"/>
   <addaction name="separator"/>
   <addaction name="actionToggleZeroEQ"/>
   <addaction name="actionPublishSelection"/>
   <addaction name="actionAutoPublishSelection"/>
   <addaction name="actionFocusOnSelection"/>
   <addaction name="actionAutoFocusOnSelection"/>
   <addaction name="actionFocusOnDisplayed"/>
   <addaction name="actionAutoFocusOnDisplayed"/>
   <addaction name="separator"/>
   <addaction name="actionRecorder"/>
  </widget>
  <action name="actionLayouts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Search and Filter</string>
   </property>
   <property name="toolTip">
    <string>Search and Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionStoredSelections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/savedselections.svg</normaloff>:/icons/savedselections.svg</iconset>
   </property>
   <property name="text">
    <string>Saved Selections</string>
   </property>
   <property name="toolTip">
    <string>Saved Selections</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionSplitHorizontally">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/horizontalSplit.svg</normaloff>:/icons/horizontalSplit.svg</iconset>
   </property>
   <property name="text">
    <string>Split &amp;horizontally</string>
   </property>
  </action>
  <action name="actionSplitVertically">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/verticalSplit.svg</normaloff>:/icons/verticalSplit.svg</iconset>
   </property>
   <property name="text">
    <string>Split &amp;vertically</string>
   </property>
  </action>
  <action name="actionKillPane">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/documentClose.svg</normaloff>:/icons/documentClose.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Kill pane</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/home.svg</normaloff>:/icons/home.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/ns_icon.png</normaloff>:/icons/ns_icon.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionShowConnectivity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/connection.svg</normaloff>:/icons/connection.svg</iconset>
   </property>
   <property name="text">
    <string>Show &amp;connectivity</string>
   </property>
   <property name="toolTip">
    <string>Show connectivity</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionShowNoHierarchyEntities">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/showNoHierarchyEntities.svg</normaloff>:/icons/showNoHierarchyEntities.svg</iconset>
   </property>
   <property name="text">
    <string>Show &amp;non-hierarchical entities</string>
   </property>
   <property name="toolTip">
    <string>Show non-hierarchical entities</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionShowEntitiesName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/showEntitiesName.svg</normaloff>:/icons/showEntitiesName.svg</iconset>
   </property>
   <property name="text">
    <string>Show &amp;entities name</string>
   </property>
   <property name="toolTip">
    <string>Show entities name</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save to NeuroML</string>
   </property>
  </action>
  <action name="actionAutoFocusOnDisplayed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/autofocusondisplay.svg</normaloff>:/icons/autofocusondisplay.svg</iconset>
   </property>
   <property name="text">
    <string>Send auto focus event on display</string>
   </property>
   <property name="toolTip">
    <string>Send focus event automatically on displayed elements</string>
   </property>
  </action>
  <action name="actionFocusOnDisplayed">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/focusondisplay.svg</normaloff>:/icons/focusondisplay.svg</iconset>
   </property>
   <property name="text">
    <string>Focus event on displayed</string>
   </property>
   <property name="toolTip">
    <string>Send focus event based on the elements displayed</string>
   </property>
  </action>
  <action name="actionFocusOnSelection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/focusonselection.svg</normaloff>:/icons/focusonselection.svg</iconset>
   </property>
   <property name="text">
    <string>Send focus on selection</string>
   </property>
   <property name="toolTip">
    <string>Send focus event based on selected elements</string>
   </property>
  </action>
  <action name="actionAutoFocusOnSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/autofocusonselection.svg</normaloff>:/icons/autofocusonselection.svg</iconset>
   </property>
   <property name="text">
    <string>Auto focus on selection</string>
   </property>
   <property name="toolTip">
    <string>Send automatically focus events based on selected elements</string>
   </property>
  </action>
  <action name="actionAutoPublishSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/autopublishselection.svg</normaloff>:/icons/autopublishselection.svg</iconset>
   </property>
   <property name="text">
    <string>Auto publish selection</string>
   </property>
  </action>
  <action name="actionPublishSelection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/publishselection.svg</normaloff>:/icons/publishselection.svg</iconset>
   </property>
   <property name="text">
    <string>Publish selection</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionToggleZeroEQ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/zeroeq.svg</normaloff>:/icons/zeroeq.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle ZeroEQ</string>
   </property>
   <property name="toolTip">
    <string>Connect or disconnect ZeroEQ</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionJSONImporter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Import JSON</string>
   </property>
  </action>
  <action name="actionCleanScene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/delete.svg</normaloff>:/icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Clean Scene</string>
   </property>
  </action>
  <action name="actionJSONExporter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Export JSON</string>
   </property>
  </action>
  <action name="actionRecorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/recorder.svg</normaloff>:/icons/recorder.svg</iconset>
   </property>
   <property name="text">
    <string>Recorder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAdvancedRecorderOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced recorder options</string>
   </property>
   <property name="toolTip">
    <string>Enable advanced recorder options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
